app/views/content/form/_attribute_field_dropdown.html.erb

```diff
@@ -0,0 +1,12 @@
+<a class='dropdown-button black-text attribute-field-dropdown' href='#' data-activates="attribute-field-menu-<%= field.id %>" data-constrainwidth='false'>
+  <i class="material-icons">more_vert</i>
+</a>
+
+<ul id="attribute-field-menu-<%= field.id %>" class='dropdown-content'>
+  <li>
+    <%= link_to "Delete this attribute forever", field,
```

Comment:
Probably should be, but [this](https://github.com/indentlabs/notebook/blob/master/app/views/content/form/_text_input.html.erb#L12) is the only place I've seen in the repo that is doing any internationalization. I took this from [actions_dropdown](https://github.com/indentlabs/notebook/blob/master/app/views/content/form/_actions_dropdown.html.erb) which isn't internationalized either. I think those types of changes should be done in a comprehensive way, I wouldn't want to dictate what helpers or patterns are defined for the implementation of i18n here.